py3status/core.py

```diff
@@ -677,6 +678,21 @@ def setup(self):
             # load and spawn i3status.conf configured modules threads
             self.load_modules(self.py3_modules, user_modules)
 
+        # determine the target output format (i3bar or tmux)
+        self.output_format = self.config["py3_config"]["general"]["output_format"]
+
+        # tmux allows configurable separators between bar entries.
+        # handle configured value or fall back to the matching default for output_format
+        self.output_format_separator = self.config["py3_config"]["py3status"].get(
+            "output_format_separator", DEFAULT_SEPARATORS[self.output_format]
+        )
+
+        # inject the value of color_separator into the separator string
+        if self.output_format_separator is not None:
+            self.output_format_separator = self.output_format_separator.format(
+                color_separator=self.config["py3_config"]["general"]["color_separator"]
```

Comment:
Does `color_separator` still exist if unspecified?

Reply:
yes, it defaults to "#333333" as per `constants.py`